src/main/java/org/opentripplanner/routing/alertpatch/TransitAlert.java

```diff
@@ -112,9 +112,10 @@ public Date getEffectiveStartDate() {
     public Date getEffectiveEndDate() {
```

Comment:
Add JavaDoc to document `null` is open ended.

Reply:
We should use Integer.MAX for open ended in this class. Preferable define a constant:

`public static final int OPEN_ENDED = Integer.MAX_VALUE; `

and use this when setting the property to open ended.

Reply:
Docs added, constant introduced.

EffectiveEndDate-calculation now also handles multiple periods where the last period is open-ended.